### PR TITLE
Revert the deprecation of some methods

### PR DESCRIPTION
**Pull request by lthurston**
Chapman and CCA vault need to be tested / compared with Solr again before this is ready for review.

### CODE REVIEW

**Comment by amywieliczka** [comment]
You can use `calisphere-id` as the field, instead of `identifier`, and instead of `_id`, so:

```hash_id.update(str(couch_doc.get('calisphere-id', 'ABC123')).encode(encoding="UTF-8"))```

This should probably actually error out noisily if there is no calisphere-id, but we can sort that later. calisphere-id should also never be an array, so you probably don't need the `str()` or `.encode()` trappings. 

```hash_id.update(couch_doc.get('calisphere-id', 'ABC123'))```

**Comment by lthurston** [comment]
@amywieliczka Ok, all changes made here.

File: metadata_mapper/mappers/mapper.py [hunk 1]
```diff
@@ -350,7 +323,7 @@ def shred(self, prop, delim=";"):
 
         return self
 
-    def copy_prop(self, prop, to_prop, skip_if_exists=[False]):
+    def copy_prop(self, prop, to_prop, no_overwrite=None, skip_if_exists=[False]):
```
Comment (amywieliczka) [inline metadata_mapper/mappers/mapper.py:326]: did you find the one case of `no_overwrite` as a parameter??? which collection was that?

**Review by amywieliczka — APPROVED** [review]

File: metadata_mapper/mappers/mapper.py (hunk 1 above)
Comment (lthurston) [reply metadata_mapper/mappers/mapper.py:326]: I didn't make a note of it, a decision I now regret. 😞

Comment (lthurston) [reply metadata_mapper/mappers/mapper.py:326]: It's here: https://registry.cdlib.org/api/v1/rikolticollection/3433/?format=json

**Pull request merged**